code/src/index.css

```diff
@@ -1,13 +1,82 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #828282;
+  margin: 0;
+  min-width: 300px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h1 {
+  font-family: 'Indie Flower', cursive;
+  font-size: 40px;
+}
+
+h2 {
+  color: #ca3e47;
+  font-family: 'Indie Flower', cursive;
+  font-size: 38px;
+  width: 100%;
+  text-align: center;
+  /* -webkit-text-stroke: 1px #fff; */
+  text-shadow: 2px 2px 0 #fff, -1px -1px 0 #fff, 1px -1px 0 #fff, -1px 1px 0 #fff, 1px 1px 0 #fff;
+}
+
+h3 {
+  font-family: 'Indie Flower', cursive;
+  font-size: 27px;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h4 {
+  font-family: 'Indie Flower', cursive;
+  font-size: 22px;
 }
+
+header img {
+  margin-top: 20px;
+  width: 600px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.form-container {
+  flex-grow: 1;
+  color: #fff;
+  font-size: 18px;
+  max-width: 600px;
+}
+
+footer {
```

Comment:
It's better to use classes for styling. It's not good practice style tags directly.